ddsrouter_test/compose/test_cases/forwarding_routes/repeater/compose.yml

```diff
@@ -0,0 +1,105 @@
+# Test description:
+#   This test checks the repeater functionality of the DDS Router. A DDS Router is deployed in the cloud with a single
+#   repeater participant. This way it is verified that the data sent from network 1 does not arrive duplicated to the
+#   subscriber of this same network, and that this same data reaches network 2.
+#
+# Test architecture:
+#
+#          ┌────────────────────────────────┐  ┌────────────────────────────┐
+#          │ cloud_edge_1_net               │  │           cloud_edge_2_net │
+#          │                                │  │                            │
+#          │                        ┌───────┴──┴───────┐                    │
+#          │                        │ ddsrouter_cloud  │                    │
+#          │                        │ (wan [repeater]) │                    │
+#          │                        └─▲──────┬──┬──────┘                    │
+#          │                          │      │  │    │                      │
+#          │           ┌──────────────┘      │  │    └─────────┐            │
+#   ┌──────┼───────────┼──────────────────┐  │  │  ┌───────────┼────────────┼──────┐
+#   │      │           │                  │  │  │  │           │            │      │
+#   │      │ ┌──────────────────┐         │  │  │  │   ┌───────▼──────────┐ │      │
+#   │      │ │ ddsrouter_edge_1 │         │  │  │  │   │ ddsrouter_edge_2 │ │      │
+#   │      │ │ (local + wan)    │         │  │  │  │   │ (local + wan)    │ │      │
+#   │      │ └───────┬────▲─────┘         │  │  │  │   └───────┬──────────┘ │      │
+#   │      │         │    │               │  │  │  │           │            │      │
+#   │      └─────────┼────┼───────────────┼──┘  └──┼───────────┼────────────┘      │
+#   │                │    │               │        │           │                   │
+#   │  ┌─────────────▼┐  ┌┴───────────┐   │        │    ┌──────▼───────┐           │
+#   │  │ sub_edge_1   │  │ pub_edge_1 │   │        │    │ sub_edge_2   │           │
```

Comment:
Why an arrow pointing towards sub_edge_1? I mean, here the samples are coming from the publisher in the same domain.

Reply:
I would just get rid of that subscriber.